$OpenBSD: patch-libgphoto2_port_gphoto2_gphoto2-port-portability_h,v 1.9 2022/01/05 10:45:06 ajacoutot Exp $

Don't define _XOPEN_SOURCE and _POSIX_C_SOURCE to old values, it hides C99
symbols like snprintf (from FreeBSD ports).

Index: libgphoto2_port/gphoto2/gphoto2-port-portability.h
--- libgphoto2_port/gphoto2/gphoto2-port-portability.h.orig
+++ libgphoto2_port/gphoto2/gphoto2-port-portability.h
@@ -83,20 +83,6 @@ typedef struct {
 
 /* yummy. :) */
 
-/* XOPEN needed for usleep */
-#ifndef _XOPEN_SOURCE
-# define _XOPEN_SOURCE 500
-#else
-# if ((_XOPEN_SOURCE - 0) < 500)
-#  undef _XOPEN_SOURCE
-#  define _XOPEN_SOURCE 500
-# endif
-#endif
-
-/* for nanosleep */
-# ifndef _POSIX_C_SOURCE
-#  define _POSIX_C_SOURCE 199309
-# endif
 # include <time.h>
 
 # include <strings.h>
